src/libConsensus/ConsensusLeader.cpp

```diff
@@ -817,16 +818,27 @@ ConsensusLeader::ConsensusLeader(
   m_commitPointMap.at(m_myID) = *m_commitPoint;
   m_commitCounter = 1;
 
-  m_allCommitsReceived = false;
+  m_sufficientCommitsReceived = false;
   m_commitRedundantCounter = 0;
   m_commitFailureCounter = 0;
   m_numSubsetsRunning = 0;
 
+  m_txnProcessingTimeout = std::max(
```

Comment:
Please add a comment here to explain that we took this value from `Node::NotifyTimeout`, and that it is meant to give backups enough time to finish processing transactions and send back the commits

Reply:
Fixed.